sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1119,6 +1264,151 @@ export class BlobClient extends StorageClient {
     });
   }
 
+  /**
+   * Set the owner, owner group or access control list for a directory.
+   *
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/update
+   *
+   * @param {string} accessControl Sets POSIX access control rights on files and directories. The value is a comma-separated list
+   *                               of access control entries. Each access control entry (ACE) consists of a scope, a type, a user
+   *                               or group identifier, and permissions in the format "[scope:][type]:[id]:[permissions]".
+   *
+   * @param {string} accessControl
+   * @param {BlobSetAccessControlOptions} [options]
+   * @returns {Promise<Models.BlobSetAccessControlResponse>}
+   * @memberof BlobClient
+   */
+  public async setAccessControl(
+    accessControl: string,
+    options: BlobSetAccessControlOptions = {}
+  ): Promise<Models.BlobSetAccessControlResponse> {
+    options.blobAccessConditions = options.blobAccessConditions || {};
+    const aborter = new ContextAborter(options.abortSignal || AbortSignal.none, DfsContext);
```

Comment:
>ContextAborter [](start = 24, length = 14)

Do we need to update other places to use the new `ContextAborter`?

Reply:
It should be fine. As other methods doesn't depend context yet.